README.md

```diff
@@ -310,6 +310,13 @@ A full changelog can be found here: [CHANGELOG.md](https://github.com/hlascelles
 
 Your [postgres](https://www.postgresql.org/) database must be at least version 9.5.0.
 
+Ruby 2.6 and above is supported. Ruby 2.5 currently still currently works but is unsupported.
```

Comment:
A bit late, but noticed you may have an extra _currently_ or _still_ in here.

Reply:
Ah, thank you. Fixed! :+1: